core/model/src/main/java/org/eclipse/rdf4j/model/vocabulary/SHACL.java

```diff
@@ -0,0 +1,822 @@
+import org.eclipse.rdf4j.model.impl.SimpleNamespace;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+
+
+/**
+ * Constants for the Shapes Constraint Language.
+ *
+ * @see <a href="https://www.w3.org/TR/2017/WD-shacl-20170303/">Shapes Constraint Language</a>
+ *
+ * @author Bart Hanssens
+ */
+public class SHACL {
+	/**
+	 * The SHACL namespace: http://www.w3.org/ns/shacl#
+	 */
+	public static final String NAMESPACE = "http://www.w3.org/ns/shacl#";
+
+	/**
+	 * Recommended prefix for the namespace: "sh"
+	 */
+	public static final String PREFIX = "sh";
+
+	/**
+	 * An immutable {@link Namespace} constant that represents the namespace.
+	 */
+	public static final Namespace NS = new SimpleNamespace(PREFIX, NAMESPACE);
+
+	// Classes
+	/** sh:AbstractResult */
+	public static final IRI ABSTRACT_RESULT;
+
+	/** sh:AndConstraintComponent */
+	public static final IRI AND_CONSTRAINT_COMPONENT;
+
+	/** sh:AndConstraintComponent-and */
+	public static final IRI AND_CONSTRAINT_COMPONENT_AND;
+
+	/** sh:BlankNode */
+	public static final IRI BLANK_NODE;
+
+	/** sh:BlankNodeOrIRI */
+	public static final IRI BLANK_NODE_OR_IRI;
+
+	/** sh:BlankNodeOrLiteral */
+	public static final IRI BLANK_NODE_OR_LITERAL;
+
+	/** sh:ClassConstraintComponent */
+	public static final IRI CLASS_CONSTRAINT_COMPONENT;
+
+	/** sh:ClassConstraintComponent-class */
+	public static final IRI CLASS_CONSTRAINT_COMPONENT_CLASS;
+
+	/** sh:ClosedConstraintComponent */
+	public static final IRI CLOSED_CONSTRAINT_COMPONENT;
+
+	/** sh:ClosedConstraintComponent-closed */
+	public static final IRI CLOSED_CONSTRAINT_COMPONENT_CLOSED;
+
+	/** sh:ClosedConstraintComponent-ignoredProperties */
+	public static final IRI CLOSED_CONSTRAINT_COMPONENT_IGNORED_PROPERTIES;
+
+	/** sh:ConstraintComponent */
+	public static final IRI CONSTRAINT_COMPONENT;
+
+	/** sh:DatatypeConstraintComponent */
+	public static final IRI DATATYPE_CONSTRAINT_COMPONENT;
+
+	/** sh:DatatypeConstraintComponent-datatype */
+	public static final IRI DATATYPE_CONSTRAINT_COMPONENT_DATATYPE;
+
+	/** sh:DerivedValuesConstraintComponent */
+	public static final IRI DERIVED_VALUES_CONSTRAINT_COMPONENT;
+
+	/** sh:DisjointConstraintComponent */
+	public static final IRI DISJOINT_CONSTRAINT_COMPONENT;
+
+	/** sh:DisjointConstraintComponent-disjoint */
+	public static final IRI DISJOINT_CONSTRAINT_COMPONENT_DISJOINT;
+
+	/** sh:EqualsConstraintComponent */
+	public static final IRI EQUALS_CONSTRAINT_COMPONENT;
+
+	/** sh:EqualsConstraintComponent-equals */
+	public static final IRI EQUALS_CONSTRAINT_COMPONENT_EQUALS;
+
+	/** sh:Function */
+	public static final IRI FUNCTION;
+
+	/** sh:HasValueConstraintComponent */
+	public static final IRI HAS_VALUE_CONSTRAINT_COMPONENT;
+
+	/** sh:HasValueConstraintComponent-hasValue */
+	public static final IRI HAS_VALUE_CONSTRAINT_COMPONENT_HAS_VALUE;
+
+	/** sh:IRI */
+	public static final IRI IRI;
+
+	/** sh:IRIOrLiteral */
+	public static final IRI IRIOR_LITERAL;
```

Comment:
Indeed, the quick&dirty tool needs some finetuning. I'll make the suggested changes